### PR TITLE
Merge code

### PR DESCRIPTION
**Pull request by fyrestone**
## What do these changes do?

Merge PRs from: https://github.com/xprobe-inc/mars

## Related issue number

Fixes #xxxx

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass, see [here](https://docs.pymars.org/en/latest/development/contributing.html#check-code-styles) for how to run them

### CODE REVIEW

File: mars/dataframe/base/cut.py [hunk 1]
```diff
@@ -362,14 +369,25 @@ def execute(cls, ctx, op):
...
 
-        cut = partial(
-            pd.cut,
-            right=op.right,
-            retbins=op.retbins,
-            precision=op.precision,
-            include_lowest=op.include_lowest,
-            duplicates=op.duplicates,
-        )
+        if pd.__version__ >= "1.1.0":
```
Comment (wjsi) [inline mars/dataframe/base/cut.py:372]: Better to use parse_version method in `mars.lib.version`.

Comment (fyrestone) [reply mars/dataframe/base/cut.py:372]: Good suggestion. I will fix it.

**Review by qinxuye — APPROVED** [review]
LGTM

**Review by zhongchun — APPROVED** [review]
LGTM.

**Pull request merged**